app/templates/more-info.hbs

```diff
@@ -1,105 +1,92 @@
-<div class="ui text container">
-  <h3 class="first headroom">More information {{help-modal model=model submit=(action "changeRoute")}} </h3>
+<div class="ui text container flow">
+  <h3 class="first headroom">More information</h3>
   <p>
     Clear licensing information and data update frequency allows for the greatest number of people to use a dataset, so please provide as much detail as possible.
   </p>
-  {{#with (changeset model InfoValidator) as |changeset|}}
```

Comment:
This part, which passes changeset to controller is needed. Please put this back.

Reply:
Oh my gosh. Sorry I didn't realize.

Reply:
Okay I know what happened. I didn't merge in the changes for the more-info page. I will fix that on Wednesday and then see if it still isn't working.